src/languageserverinstance.jl

```diff
@@ -26,13 +23,15 @@ function send(message, server)
 end
 
 function Base.run(server::LanguageServerInstance)
-    wontload_modules = []
+    loaded = []
+    wontload = []
     @schedule begin
-        for missing_module in server.user_modules
-            if !(missing_module in keys(server.cache)) && !(missing_module in wontload_modules)
-                updatecache(missing_module, server)
-                if !(missing_module in keys(server.cache))
-                    push!(wontload_modules, missing_module)
+        for (modname,uri,loc) in server.user_modules
+            if !(modname in wontload || modname in loaded) 
+                try 
+                    @eval import $modname
```

Comment:
I agree with your thoughts in #11, it's just that it's much easier to add the features and find out whats possible having everything 'just there' rather than as a three step process of retrieve/store/transfer.